shared/js/participantStore.js

```diff
@@ -119,6 +120,19 @@ loop.store.ParticipantStore = function() {
         }
       }
 
+      if (!updatedParticipant.hasNotifiedOfJoin &&
+          updatedParticipant.isHere &&
+          updatedParticipant.participantName &&
+          Date.now() - updatedParticipant.localPingTime <= PING_TIME) {
```

Comment:
There's another instance of this calculation in this file, so we should break this out to some sort of helper function.